examples/interop-examples/README.md

```diff
@@ -0,0 +1,102 @@
+# interop examples
+
+**WIP**
+
+## Run
+```
+cd examples/interop-examples
+npm i
+npm start
+```
+
+Open http://127.0.0.1:8080 in your favorite browser.
```

Comment:
There is also Safari, Edge, Internet Explorer and Opera, not everyone's favourite is Chrome or Firefox :)